src/sedcli_main.c

```diff
@@ -349,6 +395,103 @@ static void print_sed_status(int status)
 	}
 }
 
+static void sed_discv_print_normal(struct sed_opal_level0_discovery *discv, const char *dev_path)
+{
+	bool locking_enabled;
+	uint16_t comid;
+
+	locking_enabled = discv->sed_locking.locking_en ? true : false;
+	comid = discv->sed_opalv200.base_comid;
+
+	if (!comid)
+		sedcli_printf(LOG_INFO, "%s: is NOT SED-OPAL Compliant.\n", dev_path);
```

Comment:
Maybe you also indicate version of opal being supported?

Reply:
this requires addition of a new structure (opalv100). Right now I'm trying to just use the structures/feature descriptor available.
Also, I have the idea of extending this in a new pull request (Add new structures and print them in discovery command)